charts/newrelic-logging/templates/serviceaccount.yaml

```diff
@@ -1,16 +1,17 @@
-{{- if .Values.serviceAccount.create }}
+{{- if include "newrelic.common.serviceAccount.create" . -}}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  namespace: {{ .Release.Namespace }}
+  {{- if include "newrelic.common.serviceAccount.annotations" . }}
+  annotations:
+    {{- include "newrelic.common.serviceAccount.annotations" . | nindent 4 }}
+  {{- end }}
   labels:
     app: {{ template "newrelic-logging.name" . }}
     chart: {{ template "newrelic-logging.chart" . }}
     heritage: "{{ .Release.Service }}"
     release: "{{ .Release.Name }}"
-  annotations:
-  {{- if .Values.serviceAccount.annotations }}
-{{ toYaml .Values.serviceAccount.annotations | indent 4 }}
-  {{- end }}
-  name: {{ template "newrelic-logging.serviceAccountName" . }}
-{{- end -}}
+    {{- /*include "newrelic.common.labels" . | nindent 4  /!\ Breaking change /!\ */}}
```

Comment:
Hi @kang-makes , is this commented line intended to be here?

Reply:
This line is for the future self: in case you are doing a breaking change at some time, remember to change this one too to be idiomatic and create the labels using the `common-library` too.

Reply:
I see, thanks